### PR TITLE
added bftools recipe

### PR DESCRIPTION
**Pull request by ThomasWollmann**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bftools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - openjdk >=8
+
+test:
+  commands:
+    - bfconvert
```
Comment (bgruening) [inline recipes/bftools/meta.yaml]: Looks good, but the tests are failing ... maybe because this tool is again writing to stderr?

Comment (ThomasWollmann) [reply recipes/bftools/meta.yaml]: Return code is wrong. I try printing the version instead of help.

**Comment by ThomasWollmann** [comment]
@bgruening With this package we should be able to convert between the most important image formats https://docs.openmicroscopy.org/bio-formats/5.7.1/supported-formats.html

**Comment by bgruening** [comment]
Do we need bioformats then? Its only python2 and seems outdated, no?

**Comment by ThomasWollmann** [comment]
Yes, since this package contains some jars+command line wrappers. Python-bioformats contains more jars+python wrappers. We need it if we want to integrate Cellprofilter into Galaxy.

**Pull request merged**